website/src/components/misc/link.tsx

```diff
@@ -17,12 +14,18 @@ export const Link: FunctionComponent<GatsbyLinkProps<unknown>> = ({
       className={className}
       activeClassName={activeClassName}
       partiallyActive={partiallyActive}
+      ref={ref as string & React.RefObject<HTMLAnchorElement>}
```

Comment:
@rstaib  only got it working like this..

Reply:
`ref` has a quite a complex type, Its `string & React.RefObject<HTMLAnchorElement>`  I think this should work together with `React.Ref<HTMLAnchorElement>` but i really don't know. 
so this ends up being a typecast.

Reply:
Ahhh mixed things up it's regarding the type. I will take a look again later just took a short look at it...

Reply:
Ahhh mixed things up it's regarding the type. I will take a look again later just took a short look at it...